src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -436,6 +436,8 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
   const submitButtonText = useMemo(() => {
     if (isAccountsMetadataLoading) return translate('common.accountsLoading')
 
+    if (errors.manualRuneAddress?.type === 'required') return translate('RFOX.selectRuneAddress')
+
```

Comment:
Explicit handling of the `required` error, since it is not coming from a custom validation rule, but from native react-hook-form's `required` property above